cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/values/SensitiveValue.java

```diff
@@ -40,6 +43,8 @@
     @JsonIgnore
     private Serializable originalContent = null;
 
+    private boolean isExecutionPlan;
```

Comment:
this value is false by default which is the case for all other constructors
@lucian-cm WDYT?